src/mpyl/utilities/docker/__init__.py

```diff
@@ -338,3 +340,21 @@ def remove_container(logger: Logger, container: Container) -> None:
     logger.debug(f"Removing container {container.id}")
     docker.remove(container.id)
     logger.info(f"Removed container {container.id}")
+
+
+def create_ecr_repo_if_needed(logger: Logger, repo):
+    # Initialize the ECR client
```

Comment:
I think you can remove the comments in the method, it's clear what the statements are doing

Reply:
done